components/centraldashboard/public/components/main-page.js

```diff
@@ -54,11 +54,32 @@ export class MainPage extends PolymerElement {
             gettingStartedItems: {
                 type: Array,
                 value: [
-                    {text: 'Build a model in a notebook', icon: 'donut-large'},
-                    {text: 'Launch a pipeline', icon: 'donut-large'},
-                    {text: 'Deploy a solution', icon: 'donut-large'},
-                    {text: 'Compare run output', icon: 'donut-large'},
-                    {text: 'Share solution on AI Hub', icon: 'donut-large'},
+                    {
+                        text: 'Getting started with Kubeflow',
+                        desc: 'Quickly get running with your ML workflow on an existing Kubernetes installation',
+                        link: 'https://www.kubeflow.org/docs/started/getting-started/',
```

Comment:
It seems a bit odd that these links open externally versus using the iframe view which would be activated if the user clicked Kubeflow Docs from the sidenav. My open PR #2614 will refactor this section and add the router, but I'm curious if you think we should make these links appear in the iframe as well.

Reply:
Your logic makes perfect sense, and that was my initial goal. However, that is no longer the desired behavior as mentioned in #2546. 

I am waiting for your PR to then refactor all `link`s to `href`s